src/IIIFPresentation/API/Features/Storage/Requests/CreateCollection.cs

```diff
@@ -80,13 +80,28 @@ public async Task<ModifyEntityResult<PresentationCollection, ModifyCollectionTyp
             Label = request.Collection.Label,
             Parent = parentCollection.Id,
             Slug = request.Collection.Slug,
-            Thumbnail = request.Collection.Thumbnail,
             Tags = request.Collection.Tags,
             ItemsOrder = request.Collection.ItemsOrder
         };
         
-        await using var transaction = 
-            await dbContext.Database.BeginTransactionAsync(cancellationToken);
+        string? convertedIIIFCollection = null;
+
+        if (!request.Collection.Behavior.IsStorageCollection())
+        {
+            try
+            {
+                convertedIIIFCollection = ConvertToIIIFCollection(request, collection);
+            }
+            catch (Exception ex)
+            {
+                logger.LogError(ex, "Error attempting to validate collection is IIIF");
```

Comment:
This message feels a little unclear to me - would something like "An error occurred while attempting to validate the collection as IIIF" be easier to read?